src/main.rs

```diff
@@ -79,7 +79,7 @@ struct SelectionResult {
 }
 
 fn get_starting_location() -> Location {
-    let mut location = Location::Session;
+    let mut location = get_default_tab();
     if get_current_session_id() == "" {
         location = Location::Directory;
```

Comment:
This `get_current_session_id` check probably should only apply if location is already `Session`. Otherwise it can override the default tab.